### ISSUE TITLE
Accessibility Refactor

### ISSUE THREAD

**Issue by corwintines** [ethereum/clrfund#295]
There are a number of portions in our application that lack accessibility functionality that should get added. We should refactor these to bring this functionality in. This issue is a placeholder, so please add to the list as you come across them:
- []
